packages/eslint-config-4catalyzer-react/rules.js

```diff
@@ -21,9 +11,6 @@ module.exports = {
       depth: 25,
     },
   ],
-  'jsx-a11y/label-has-for': 'off',
-  'react-hooks/rules-of-hooks': 'error',
-  'react-hooks/exhaustive-deps': 'warn',
```

Comment:
this is an error upstream, but we don't allow errors in our pre-commit hooks mostly, so who cares

Reply:
only frontend does the no warnings thing, which isn't tenable for telemed right now, it's got a lot of warning that are intentionally left there (e.g. alerts, console.logs)

Reply:
okay, rather, it gets auto-fixed, so we still need to forcibly pragma it